package/MDAnalysis/coordinates/XDR.py

```diff
@@ -120,7 +120,8 @@ class XDRBaseReader(base.ReaderBase):
        XDR offsets read from trajectory if offsets file read-in fails
     .. versionchanged:: 2.0.0
        Add a InterProcessLock when generating offsets
-
+    .. versionchanged:: 2.4.0
+       Use a direct read into ts attributes
```

Comment:
Although this is simply in the TRR and XTC xdr reader classes, it concur that we should keep a note here in the user-visible docs.